File: Q7.py
eleitores=int(input("Digite o total de eleitores no município: "))
brancos=int(input("Digite o total de votos brancos: "))
nulos=int(input("Digite o total de votos nulos: "))
válidos=int(input("Digite o total de votos válidos: "))
percentual_brancos=brancos*100/eleitores
percentual_nulos=nulos*100/eleitores
percentual_válidos=válidos*100/eleitores
print("O percentual de votos, em relação ao total de eleitores no município, é:")
print(f"{percentual_brancos:.1f}% votos brancos, {percentual_nulos:.1f}% votos nulos e {percentual_válidos:.1f}% votos válidos.")

File: Q7.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    float eleitores, brancos, nulos, validos, percentual_brancos, percentual_nulos, percentual_validos;
    cout << "Digite o total de eleitores no município: ";
    cin >> eleitores;
    cout << "Digite o total de votos brancos: ";
    cin >> brancos;
    cout << "Digite o total de votos nulos: ";
    cin >> nulos;
    cout << "Digite o total de votos válidos: ";
    cin >> validos;
    percentual_brancos = brancos * 100 / eleitores;
    percentual_nulos = nulos * 100 / eleitores;
    percentual_validos = validos * 100 / eleitores;
    cout << fixed << setprecision(1);
    cout << "O percentual de votos, em relação ao total de eleitores no município, é:" << endl;
    cout << percentual_brancos << "% votos brancos, ";
    cout << percentual_nulos << "% votos nulos e ";
    cout << percentual_validos << "% votos válidos." << endl;
    return 0;
}